source/extension-manifest.json

```diff
@@ -207,13 +207,23 @@
                         "resultSelector": "jsonpath:$.[*]"
                     },
                     {
-                        "name": "OctopusListProjects",
-                        "endpointUrl": "$(endpoint.url)/api/projects?partialName=$(PartialNameFilter)",
+                        "name": "OctopusAllProjectGroups",
```

Comment:
Grabbing all project groups, following the existing pattern of "bind this whole data source to a dropdown" approach from this archaic library.